### PR TITLE
Mention workarounds for 25MB file size limit

### PR DESCRIPTION
**Pull request by mcraiha**
Since this can be a very common issue with apps that have many dependencies

### CODE REVIEW

File: products/pages/src/content/framework-guides/deploy-a-blazor-site.md [hunk 1]
```diff
@@ -73,6 +73,12 @@ For the complete guide to deploying your first site to Cloudflare Pages, refer t
 After deploying your site, you will receive a unique subdomain for your project on `*.pages.dev`. 
 Every time you commit new code to your Blazor site, Cloudflare Pages will automatically rebuild your project and deploy it. You will also get access to [preview deployments](/platform/preview-deployments) on new pull requests, so you can preview how changes look to your site before deploying them to production.
 
+## Troubleshooting
+
+### A file is over the 25MB limit
+
+If you get error like *"Error: Asset "/opt/buildhome/repo/output/wwwroot/_framework/dotnet.wasm" is over the 25MB limit"* then you have two options. Either you try to reduce size of your assets with following [guide](https://docs.microsoft.com/en-us/aspnet/core/blazor/performance?view=aspnetcore-6.0#minimize-app-download-size) **or** remove the _*.wasm_ files from the output (e.g. `rm output/wwwroot/_framework/*.wasm`) and modify your Blazor app to [load the Brotli compressed files](https://docs.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/webassembly?view=aspnetcore-6.0#compression) instead.
```
Comment (deadlypants1973) [inline products/pages/src/content/framework-guides/deploy-a-blazor-site.md]:
```suggestion
If you receive the error message `Error: Asset "/opt/buildhome/repo/output/wwwroot/_framework/dotnet.wasm" is over the 25MB limit`, you have two options:

1. Reduce the size of your assets with the following [guide](https://docs.microsoft.com/en-us/aspnet/core/blazor/performance?view=aspnetcore-6.0#minimize-app-download-size).

Or

2. Remove the `*.wasm` files from the output (`rm output/wwwroot/_framework/*.wasm`) and modify your Blazor application to [load the Brotli compressed files](https://docs.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/webassembly?view=aspnetcore-6.0#compression) instead. 
```

**Comment by CEOSPFUTURE2** [comment]
Hello my name is warith akbar my name  warith akbar . I'm the owner of fact
News Network and owner of S&P 500 future.   I need a tec to my headquarters
asap to help secure my company.

**Comment by mcraiha** [comment]
Any progress on this one?

**Comment by WalshyDev** [comment]
This is a very specific case. If this was much more generalized I think it would be better. The chance of someone going to this section and having this _one specific_ setup is unlikely.

---

I'd suggest changing this to something like:

### A file is over the 25MB limit

If you receive a similar error message to `Error: Asset "/opt/buildhome/repo/some_file" is over the 25MB limit`, there are a few things you could do:

1. Check if the file is used in your website
  - If the file is not being used for a website (possibly a build dependency or similar) then you will want to not have this file in the build output folder. You may want to re-configure your root and build output folders or move the file out once built
1. Try to reduce the file size
  - If something like an image is hitting the file limit you could do lossless compression in an attempt to get it under the limit
1. Check if your framework/library has any additional tips below

#### Blazor
If you hit the file limit on `.wasm` files, you have two options:

1. Reduce the size of your assets with the following [guide](https://docs.microsoft.com/en-us/aspnet/core/blazor/performance?view=aspnetcore-6.0#minimize-app-download-size).

Or

2. Remove the `*.wasm` files from the output (`rm output/wwwroot/_framework/*.wasm`) and modify your Blazor application to [load the Brotli compressed files](https://docs.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/webassembly?view=aspnetcore-6.0#compression) instead.

**Comment by mcraiha** [comment]
I can redo the patch if you are doing generic troubleshooting steps for all platforms. There are other Blazor related troubleshooting steps I would like to write down, but I can wait with those.

**Comment by jkup** [comment]
Hey @mcraiha!

Thanks for this! And thanks @WalshyDev too!

@mcraiha can you quickly help me understand a bit more? I've never used Blazor. Am I paraphrasing correctly when I say "Blazor has the option to generate and serve WASM files. However, these WASM files can quickly get too large and exceed Cloudflare's 25mb limit. When Blazor generates WASM files, it also generates compressed versions, both Gzip and Brotli. If you exceed the 25mb limit, you can save space by deleting the WASM and Gzip versions of your application, and configuring your site to serve the Brotli versions directly."

**Comment by mcraiha** [comment]
@jkup basically that. The Blazor WebAssembly mode (the other mode is Blazor Server, but it is not related to this) will always output WASM+JS+HTML, and (by default) files generated to the *_framework* folder include non compressed one (e.g. dotnet.wasm), GZ compressed one (e.g. dotnet.wasm.gz) and Brotli compressed one (e.g. dotnet.wasm.br).

**Review by jkup — APPROVED** [review]
Sorry this took me so long to get back to! I think it's good since it's in the Blazor guide itself!

**Pull request merged**